File: CSharp/LeaderBoard/SelfDrivingCar.cs
namespace TDDMicroExercises.LeaderBoard
{
    public class SelfDrivingCar : Driver
    {
        public SelfDrivingCar(string algorithmVersion, string company) : base(algorithmVersion, company)
        {
            AlgorithmVersion = algorithmVersion;
        }

        public string AlgorithmVersion { get; set; }
    }
}

File: php/src/Leaderboard/SelfDrivingCar.php
<?php

declare(strict_types=1);

namespace RacingCar\Leaderboard;

class SelfDrivingCar extends Driver
{
    public function __construct(
        public string $algorithmVersion,
        string $company
    ) {
        parent::__construct(null, $company);
    }
}
